WolvenKit.App/ViewModels/Shell/ChunkViewModel.cs

```diff
@@ -10,14 +11,15 @@
 using System.Runtime.Serialization;
 using System.Text.Json;
 using System.Threading.Tasks;
-using System.Windows;
+using System.Windows.Forms;
```

Comment:
why do we need a dependency on Winforms here?

Reply:
Rider auto-corrected the existing `System.Windows` to `System.Windows.Forms`, narrowing down the dependency. Awfully nice of it :D